.travis.yml

```diff
@@ -7,13 +7,11 @@ env:
   global:
   - DOCKER_REPO=nhsuk/${REPLACEME}
   - DOCKER_USERNAME=nhsukautomata
-  - RANCHER_ACCESS_KEY=655696CC96424E6D4DB0
-  - RANCHER_ENVIRONMENT=c2s-dev
   - RANCHER_TEMPLATE_NAME=${REPLACEME}
-  - RANCHER_URL=https://rancher.nhschoices.net
+  - RANCHER_URL=${URL-https://rancher.nhschoices.net}
```

Comment:
What is the rational for this? I guess it's for testing like RANCHER_TEMPLATE_BRANCH_NAME but it's not mentioned in the README.